docs: add more details and sse/ws comparison (#1126)
dotansimha/graphql-yoga: website/docs/features/subscriptions.mdx

## Subscriptions over WebSockets with `graphql-ws`

> Suppose you want to enable [`graphql-transport-ws`](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md) protocol support with GraphQL Yoga. In that case, we can easily integrate GraphQL Yoga and `graphql-ws` libraries by using the examples in [Envelop](https://envelop.dev) and [graphql-ws](https://github.com/enisdenjo/graphql-ws) integration docs.

Author: ```suggestion
Suppose you want to use the [`graphql-transport-ws`](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md) protocol with GraphQL Yoga, you can use the [`graphql-ws`](https://github.com/enisdenjo/graphql-ws) library.
```
